neurolib/models/multimodel/builder/base/backend.py

```diff
@@ -220,6 +297,20 @@ def run(self, duration, dt, noise_input, **kwargs):
                 argdefs=(),
             )
         )
+        self.compiled_function = integrate
+
+    def run(self, duration, dt, noise_input, symbol_params, float_params, **kwargs):
```

Comment:
this is the new run function - compilation is elsewhere - this assumes that the compilation is already done